packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -22,21 +22,22 @@ const GlobalClassNames = {
 
 export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
   const { className, titleLabelClassName, theme, vertical, disabled, showTransitions, showValue } = props;
-  const { palette } = theme;
+  const { palette, semanticColors } = theme;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
 
   // Tokens
-  const sliderInteractedActiveSectionColor = palette.themePrimary;
-  const sliderInteractedInactiveSectionColor = palette.themeLight;
-  const sliderRestActiveSectionColor = palette.neutralSecondary;
-  const sliderRestInactiveSectionColor = palette.neutralTertiaryAlt;
+  const sliderInteractedActiveSectionColor = semanticColors.inputBackgroundCheckedHovered;
+  const sliderHoverSectionColor = semanticColors.inputBackgroundChecked;
+  const sliderInteractedInactiveSectionColor = semanticColors.inputBackgroundCheckedAlt;
+  const sliderRestActiveSectionColor = semanticColors.smallInputBorder;
+  const sliderRestInactiveSectionColor = semanticColors.disabledBorder;
 
   const sliderDisabledActiveSectionColor = palette.neutralTertiary;
   const sliderDisabledInactiveSectionColor = palette.neutralLight;
 
-  const sliderThumbBackgroundColor = palette.white;
-  const sliderThumbBorderColor = palette.neutralSecondary;
-  const sliderThumbDisabledBorderColor = palette.neutralTertiaryAlt;
+  const sliderThumbBackgroundColor = semanticColors.inputBackground;
```

Comment:
The thumb itself is not an input so I'm wondering if this semantically makes sense... can we use another slot or does it makes sense to use input slots for the different parts of slider that aren't inputs themselves.

Reply:
it follows input colors though, you want the colors here to be similar to those of the toggles, radio buttons, and checkboxes that it will be sitting next to in a form